build/props/common.props

```diff
@@ -1,50 +1,94 @@
 <!--suppress CheckTagEmptyBody -->
-<Project>
+<Project InitialTargets="SilkAddIcon;SilkGenerateNuGetReadme">
     <PropertyGroup>
-        <PackageIconUrl>https://static.ultz.co.uk/img/SilkDotNet.png</PackageIconUrl>
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
         <LangVersion>preview</LangVersion>
         <Authors>.NET Foundation and Contributors</Authors>
         <PackageReleaseNotes>
-            Silk.NET October 2021 Update
-
-            - Add bindings for DirectX Video Acceleration (DXVA)
-            - Add bindings for D3D9Ex
-            - Add bindings for extra core Win32 APIs (as required by DXVA)
-            - Add a .NET 6 code path for GlobalMemory using NativeMemory
-            - Add more TFMs in addition to .NET Standard 2.0 for DirectX bindings 
-            - Add enhanced anonymous struct support with ref-returning properties on .NET Standard 2.1 and up
-            - Update to Vulkan 1.2.194
-            - Update to latest OpenCL specifications
-            - Fix MarshalDirectiveException when a struct containing a char passes across a native boundary
-            - Fix Vulkan out parameters not having their structure type set before passing across the native boundary
-            - Fix native libraries not being found in some cases for self-contained executables
+            Silk.NET November 2021 Update
+            
+            - Added support for BigInteger and Complex in the Silk.NET.Maths.Scalar APIs.
```

Comment:
Should we add a thanks to @WhiteBlackGoose here? Feel like that'd be deserved, we've not done so in the past, but it'd be a nice gesture for the future if we start putting (#PR thank you @<user>) from now on?

Reply:
Sure.

Reply:
Done.